lib/firebase_constants.dart

```diff
@@ -23,3 +24,4 @@ String get appId => _constants['appId'];
 String get measurementId => _constants['measurementId'];
 List<String> get allowedEmailDomains =>
     _constants['allowedEmailDomains'].split('|');
+String get metadataPath => _constants['metadataPath'];
```

Comment:
let's not store the path to config in the firebase constants - it's fine if this is hardcoded in the code directly, as would be the path for the interactions. Unless you expect that this will change independent of the code?

Reply:
on second thought, I see now that it's a bit different for interactions, because you're reading that path out of the config. Let's talk more outside this review about what would be the right way of pointing this reporting tool at different dataset.

Reply:
Marking as resolved at the moment, tracking this in here: https://github.com/larksystems/katikati-reporting/issues/8